docs/data/date-pickers/date-range-field/date-range-field.md

```diff
@@ -8,7 +8,7 @@ packageName: '@mui/x-date-pickers-pro'
 
 # Date Range Field [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
-<p class="description">The Date Range Field components let the user select a date range with the keyboard.</p>
```

Comment:
@LukasTy if you prefer I can keep "components let" for pages where we have several components displayed (range fields and clock), but I favored the wording consistency since the other pages also have several components (the variants).